src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -11,21 +11,21 @@
         <v-flex xs4>
           <label class="d-flex flex-wrap">
             <span class="mr-2">Mailing Address</span>
-            <v-chip v-if="mailingChanged"
+            <v-chip v-if="this.isCorrectionView() && mailingChanged"
```

Comment:
Within the template, it isn't required that you refer to this method with `this`.

Reply:
Yes, you're right. Thanks!